payment_allpay/README.md

```diff
@@ -0,0 +1,6 @@
+# payment_allPay
```

Comment:
Hi @jeffery9 please use standard README.rst template (https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)
There is no md file in OCA repos